src/map/clif.c

```diff
@@ -415,248 +416,264 @@ bool clif_send(const void* buf, int len, struct block_list* bl, enum send_target
 	sd = BL_CAST(BL_PC, bl);
 
 	switch(type) {
+	case ALL_CLIENT: //All player clients.
```

Comment:
style fixes like this should be in separate commit. this can help players to apply own patches to server code